.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I think .vscode should be gitignored

Reply:
I think it's useful for ppl who use vscode

Reply:
u very right

Reply:
@aadibajpai good for consistency in some cases I think but my settings have some personal prefs that might not need to get pushed around

Reply:
ah ok I think for vaken they maintain it